deploy/crds/kieapp.crd.yaml

```diff
@@ -97,45 +93,261 @@ spec:
                 server:
                   type: object
                   description: Configuration of the RHPAM KIE server
-#                  anyOf:
-#                  - required:
-#                    - env
-#                  - required:
-#                    - resources
                   properties:
-                    env:
-                      type: array
-                      items:
+                    deployments:
+                      type: integer
+                      format: int16
+                      description: Number of Server sets that will be deployed
+                    from:
+                      type: object
+                      description: Image definition to use for all the servers
+                      required:
+                        - kind
+                        - name
+                      properties:
+                        kind:
+                          type: string
+                          description: Object kind. e.g. ImageStreamTag
+                        name:
+                          type: string
+                          description: Object name
+                        namespace:
+                          type: string
+                          description: Namespace where the object is located
+                    build:
```

Comment:
This is to create multiple KIE server sets that are all built from the same git source? I don't think we need this. I suggest removing this, and if they want to do a single s2i build, they can provide a servers array of size one with that in there.